pkg/client/unversioned/request.go

```diff
@@ -737,9 +770,8 @@ func (r *Request) request(fn func(*http.Request, *http.Response)) error {
 	}
 }
 
-// Do formats and executes the request. Returns a Result object for easy response
+// Do formats and executes the request.  Returns a Result object for easy response
```

Comment:
Please avoid needless whitespace and doc changes as they distract from the actual code.